file.go

```diff
@@ -0,0 +1,205 @@
+package cfile
+
+// TODO:
+// * explore file locking
```

Comment:
Are you referring to OS-level locking? My only reservation about that is keeping the "purity" of the golang application. What I mean is, depending on the OS as little as possible. That way, if some unikernels gain lots of traction in the future, it will be a more seamless integration.

Reply:
Yeah I just wanted to look into it, didn't have any specific plans, yet.